tests-ng/util/update_runtime.sh

```diff
@@ -0,0 +1,325 @@
+#!/usr/bin/env bash
```

Comment:
isn't it a bit ironic to have this non trivial bash program to keep python updated? any reason not do implement this in python?

Reply:
Also asked me that question but went with bash as currently everything "build" related in `tests-ng/util` is bash.
Not sure if it would be way less "non trivial" in python :shrug: but would not care to re-write it.
@nkraetzschmar What would be your opinion on that?

Reply:
Just checked, could provide a ~220 lines python script instead of the ~320 lines bash script, but the "issue" of having an additional requirements file (different purpose) remains.